packages/paste-core/components/card/stories/index.stories.tsx

```diff
@@ -1,11 +1,34 @@
 import * as React from 'react';
 import {storiesOf} from '@storybook/react';
-import {action} from '@storybook/addon-actions';
-import {Card, CardTitle, CardBody} from '../src';
+import {Heading} from '@twilio-paste/heading';
+import {Text} from '@twilio-paste/text';
+import {Card, CardFooter} from '../src';
 
-storiesOf('Components|Card', module).add('Default', () => (
-  <Card onClick={action('clicked')}>
-    <CardTitle>Title</CardTitle>
-    <CardBody>Body</CardBody>
-  </Card>
-));
+storiesOf('Components|Card', module)
+  .add('Default', () => (
+    <Card>
```

Comment:
Would it be possible to hook up the first story to be able to modify the props with the knobs feature of storybook?

Kind of like https://github.com/twilio-labs/paste/blob/master/packages/paste-core/utilities/text/stories/index.stories.tsx#L68